### DISCUSSION TITLE
Is there a cache plugin?

### DISCUSSION THREAD

**Discussion by idefixxx** [datenstrom/community#185] [Ask a question] [answered]
Hi,
I need a cache plugin for the preview blogpage. But I do not know how to program it.
Any ideas how I can save this page?

Regards Idefixxx

**Comment by markseuffert** [comment, marked as answer]
Need a faster website... you can add individual web pages to the cache.
https://github.com/annaesvensson/yellow-command#how-to-build-a-static-cache

Is this what you are looking for?
